release/tasks/disk-uploader/README.md

```diff
@@ -12,7 +12,7 @@ When user runs [KubeVirt Tekton Tasks](https://github.com/kubevirt/kubevirt-tekt
 
 ### Parameters
 
-- **EXPORT_SOURCE_KIND**: The name of the export source kind
+- **EXPORT_SOURCE_KIND**: Specify the export source kind (Expected values `vm`, `vmsnapshot`, or `pvc`)
 - **EXPORT_SOURCE_NAME**: The name of the export source
```

Comment:
I believe we should mention push_timeout is optional in default value in readme

Reply:
Ok, mentioned it.